cpp/ps0001/main.cpp

```diff
@@ -4,17 +4,21 @@ using namespace std;
 
 int main()
 {
+  // prompting user for 3 numers to add 1/2
+  unsigned short num1;
+  cout << "enter value of a " <<endl;
+  cin >> num1;
+  unsigned short num2;
+  cout << "enter value of b " <<endl;
+  cin >> num2;
+  unsigned short num3;
+  cout << "enter value of c " <<endl;
+  cin >> num3;
 
-int a;
-cout <<"enter value of a "<<endl;
-cin >> a ;
-int b;
-cout <<"enter value of b "<<endl;
-cin >> b ;
-int c;
-cout <<"enter value of c "<<endl;
-cin >> c ;
-int sum= a+b+c;
-cout <<"sum= "<< sum<<endl;
-return 0;
+  // outputting the result of the summtion process 3/3
+  unsigned short sum = num1 + num2 + num3;
+  cout << "sum= " << sum << endl;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as program is user-friendly